### PR TITLE
Register carl.is-a.dev

### PR DESCRIPTION
**Pull request by drichardcarl**
Register carl.is-a.dev with A record pointing to 62.169.21.91.

### CODE REVIEW

File: domains/carl.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "drichardcarl",
+           "email": "",
+           "discord": "355219910017810473",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.c7N6MxjXTZENy08srgl-PDV-GuqNckghd3YJLjldUAAo80DJfLb2_CNzX14C7K04yjU5atIrQwd3hMLE8fhr5zwqz0hZGKq8AMqYW0fVlKxXXf2Gl9pX3YkF-BoZs9FU3zZc5Fk85wpSLJU5FRx1p4cVab819_gaREhOKHGFVrMq-WQsyqxFzVGvcDadXgyAYgIT6p1ajNi5dB2Xs_Ep9beNEKlxSvzx-DRdv_cCwyNZ_cARYIQWXaTt2h1wPPubcHmmZwomjqk9pMNj_wa5S94OSQegOM8Y85LlS1XQSdNg65nLRhszZmJPAqYv2bcKsoNeGINTiRbW4IsIwRlZDg.zwNWRe_zwSYGwyaZd8wZuQ.BKje0WZPqcRTjq4ruCEp4E1lTJQQcPikzLPbZRPIKs-La8elO4P3io9GTZ8ULiF09-x9-NwBsyy2RaRLkKNBlRmjOzeME2O00aJFcoc88a0.BmVKm-Hy66ImwjsWOh5ALw"
+        },
+    
+        "record": {
+            "CNAME": "62.169.21.91"
```
Comment (DEV-DIBSTER) [inline domains/carl.json:10]: CNAME should be hostnames not IPs. Please read the docs at https://docs.is-a.dev before making a PR.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

File: domains/carl.json (hunk 1 above)
Comment (drichardcarl) [reply domains/carl.json:10]: Sorry, should have been A record, I'll decline this and open another PR. Thanks.

Comment (drichardcarl) [reply domains/carl.json]: All good now @DEV-DIBSTER , thanks.

**Comment by drichardcarl** [comment]
> This site is inaccessible, please provide a preview.

Hi @orxngc , is that a requirement before approving this PR?
I already have the VPS instance, I just haven't set it up yet.

Thanks!

image

**Comment by orxngc** [comment]
> is that a requirement before approving this PR?

yes, it's a requirement, sorry

**Pull request closed without merging**